packages/integration-karma/test/rendering/slotting/index.spec.js

```diff
@@ -3,6 +3,7 @@ import { createElement } from 'lwc';
 import RenderCountParent from 'x/renderCountParent';
 import FallbackContentReuseParent from 'x/fallbackContentReuseParent';
 import RegressionContainer from 'x/regressionContainer';
+import FallbackContentReuseDynamicKeyParent from 'x/fallbackContentReuseDynamicKeyParent';
```

Comment:
I wanted to be extra-sure that this still worked even if we don't scope keys coming from `api_key`, so I added a test. As it turns out, `api_key` is already inherently scoped.